projects/contributor-experience.md

```diff
@@ -0,0 +1,114 @@
+# Contributor Experience
+
+## Background and description
+
+Over the last few years the OpenTelemetry project has seen massive growth, not
+only in adoption and lines of code, but also in the number of individuals who
+contribute to this project in different ways.
+
+While in the beginning of the project contributors knew each other well and
+could share implicit in-group knowledge effortlessly, the project has outgrown that
+phase a while ago and we see different kinds of problems, that make it hard
+for long-term contributors to maintain the project sustainably and for new-comers
+to enter the project effortlessly.
+
+In this project proposal we suggest to form a cross-cutting special interest group
+that takes responsibility for driving contributor experience projects and that
+closely collaborates with all other SIGs and governing bodies to increase contributor
+sustainability, velocity and happiness.
+
+### Current challenges
+
+Among others the challenges the OpenTelemetry project is facing in regards of
+contributor experience are:
+
+* No consistent way of moving contributors "up the ladder", e.g. from member to
+  approver, from approver to maintainer etc, which makes those changes invisible
+  to the larger project and opaque to project leadership on who is doing
+  what.
+* No clear onboarding path for new contributors, or contributors taking on new
+  responsibilities within the project (triagers, approvers, maintainers, committee
+  members), which leads to friction in understanding of the overall project goals,
+  to misalignment in implementations and the feeling of contributors being remote
+  to other parts of the project.
+* No explicit contributor guidebook that helps contributors to answer common
+  questions no matter where they are on the contributors' ladder.
+* Mentorship for new contributors or contributors stepping up into new roles to
+  make them part of the project and aware of.  
+* Only implicit and unstructured recognition for contributors, which would help
+  contributors to feel seen and recognized.
```

Comment:
```suggestion
* Only implicit and unstructured recognition for contributors, instead of explicit
   recognition which would help contributors to feel seen and recognized.
```